setup.py

```diff
@@ -12,14 +12,14 @@ def get_packages(package):
             if os.path.exists(os.path.join(dirpath, '__init__.py'))]
 
 setup(name='zc_common',
-      version='0.1.0',
+      version='0.1.1',
```

Comment:
Can you update the `README` installation instructions to reference the new tag too?